source/core/owt_base/internal/TransportServer.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) <2019> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TransportServer_h
+#define TransportServer_h
+
+#include "IOService.h"
+#include "RawTransport.h"
+#include "TransportBase.h"
+
+#include <boost/asio.hpp>
+#include <boost/scoped_ptr.hpp>
```

Comment:
only include those boost header files used by .h;  if some are used by impl, you can include them there. scoped_ptr and some other are the example.